portfolio/src/main/webapp/script.js

```diff
@@ -26,3 +26,31 @@ function addRandomGreeting() {
   const greetingContainer = document.getElementById('greeting-container');
   greetingContainer.innerText = greeting;
 }
+
+function addRandomQuote() {
+
+  const quotes =
+      ['We cannot change the cards we are dealt, just how we play the hand.', 
+      'When it comes to men who are romantically interested in you, it’s really simple. Just ignore everything they say and only pay attention to what they do.', 
+      'The key question to keep asking is, Are you spending your time on the right things? Because time is all you have.', 
+      'Showing gratitude is one of the simplest yet most powerful things humans can do for each other.'];
+    
+//    const quotes = ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+
+  // Pick a random greeting.
+  const quote = quotes[Math.floor(Math.random() * quotes.length)];
+
+  // Add it to the page.
+  const quoteContainer = document.getElementById('quote-container');
+  quoteContainer.innerText = quote;
+}
+
+/* Toggle between adding and removing the "responsive" class to topnav when the user clicks on the icon */
+function navFunction() {
```

Comment:
This is not a good naming for function, as it's already obviously a function. Maybe `onIconClicked()`

Reply:
makes sense, thanks for the advice. changed, thanks